Bootstrapping the project via quarkus-maven-plugin is broken

**Describe the bug**
When a Quarkus Bootstrapping project(1.3.2.Final) is created via quarkus-maven-plugin, the project can't be built. The error logs seem to bump 1.4.1.Final version.

**Expected behavior**
Build Success

**Actual behavior**
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.4.1.Final:build (default) on project getting-started: Failed to build quarkus application: Failed to load current project at /tmp/hello/getting-started -> [Help 1]

**To Reproduce**
Steps to reproduce the behavior:
1. Create a new Bootstrapping project:
mkdir /tmp/hello && cd /tmp/hello && \
mvn io.quarkus:quarkus-maven-plugin:1.3.2.Final:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=getting-started \
    -DclassName="org.acme.quickstart.GreetingResource" \
    -Dpath="/hello"
2. mvn -f /tmp/hello/getting-started clean package -Pnative -DskipTests

**Configuration**
```properties
# Add your application.properties here, if applicable.

```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)
See the attachment

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: 
Linux workspaceggmc3rv2lscexlru.quarkus-tools-b4ccf469d-jszbq 4.18.0-147.8.1.el8_1.x86_64 #1 SMP Wed Feb 26 03:08:15 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

 - Output of `java -version`: 
openjdk version "1.8.0_242"
OpenJDK Runtime Environment (build 1.8.0_242-b08)
OpenJDK 64-Bit Server VM (build 25.242-b08, mixed mode)

 - GraalVM version (if different from Java): 
/usr/local/graalvm-ce-java8-19.3.1

 - Quarkus version or git rev: 
1.3.2.Final

 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
Apache Maven 3.6.3 

**Additional context**
<img width="1636" alt="error" src="https://user-images.githubusercontent.com/14066977/80263200-ddd49b00-865d-11ea-9c0a-00eef199a4ea.png">

(Add any other context about the problem here.)

--- a/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/BootstrapMavenContext.java
+++ b/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/BootstrapMavenContext.java
@@ -244,7 +244,7 @@ public class BootstrapMavenContext {
         try {
             return LocalProject.loadWorkspace(this);
         } catch (BootstrapException e) {
-            throw new AppModelResolverException("Failed to load current project at " + getCurrentProjectPomOrNull());
+            throw new AppModelResolverException("Failed to load current project at " + getCurrentProjectPomOrNull(), e);
         }
     }
 
@@ -635,11 +635,7 @@ public class BootstrapMavenContext {
         if (alternatePomName != null) {
             alternatePom = Paths.get(alternatePomName);
             if (alternatePom.isAbsolute()) {
-                Path pom = alternatePom;
-                if (Files.isDirectory(pom)) {
-                    pom = pom.resolve("pom.xml");
-                }
-                return Files.exists(pom) ? pom : null;
+                return pomXmlOrNull(alternatePom);
             }
         }
 
@@ -684,24 +680,23 @@ public class BootstrapMavenContext {
 
         // we are not in the context of a Maven build
         if (alternatePom != null && alternatePom.isAbsolute()) {
-            return alternatePom;
+            return pomXmlOrNull(alternatePom);
         }
 
         // trying the current dir as the basedir
         final Path basedir = Paths.get("").normalize().toAbsolutePath();
         if (alternatePom != null) {
-            Path pom = basedir.resolve(alternatePom);
-            if (Files.exists(pom)) {
-                if (Files.isDirectory(pom)) {
-                    pom = pom.resolve("pom.xml");
-                    return Files.exists(pom) ? pom : null;
-                }
-                return pom;
-            }
-            return null;
+            return pomXmlOrNull(basedir.resolve(alternatePom));
         }
         final Path pom = basedir.resolve("pom.xml");
         return Files.exists(pom) ? pom : null;
+    }
+
+    private static Path pomXmlOrNull(Path path) {
+        if (Files.isDirectory(path)) {
+            path = path.resolve("pom.xml");
+        }
+        return Files.exists(path) ? path : null;
     }
 
     public Path getCurrentProjectBaseDir() {

